Library/ViewModels/CommentCellViewModel.swift

```diff
@@ -11,23 +11,26 @@ public protocol CommentCellViewModelInputs {
 }
 
 public protocol CommentCellViewModelOutputs {
-  /// Emits author's badge for a comment
+  /// Emits author's badge for a comment.
   var authorBadge: Signal<Comment.AuthorBadge, Never> { get }
 
-  /// Emits a url to the comment author's image
+  /// Emits a url to the comment author's image.
   var authorImageURL: Signal<URL, Never> { get }
 
-  /// Emits text containing author's fullname or username
+  /// Emits text containing author's fullname or username.
   var authorName: Signal<String, Never> { get }
 
-  /// Emits text containing comment body
+  /// Emits text containing comment body.
   var body: Signal<String, Never> { get }
 
-  /// Emits text  relative time the comment was posted
+  /// Emits text  relative time the comment was posted.
   var postTime: Signal<String, Never> { get }
 
-  /// Emits a Bool determining if the reply and flag buttons in the bottomColumnStackView are hidden
+  /// Emits a Bool determining if the reply and flag buttons in the bottomColumnStackView are hidden.
   var replyButtonIsHidden: Signal<Bool, Never> { get }
+
+  /// Emits whether or not the view replies stack view is hidden.
+  var viewRepliesStackViewHidden: Signal<Bool, Never> { get }
```

Comment:
If this is emitting a Bool we should probably follow the Swift convention of the name beginning with `is`